widgets/src/types/rowType.js

```diff
@@ -1,21 +1,29 @@
-import { shape, string, number, object } from 'prop-types';
+import { shape, string } from 'prop-types';
 
 export default shape({
-  inlimit: string,
-  application: string,
-  incdownpayment: string,
-  id: number,
+  id: string,
   name: string,
-  subject: string,
-  desc: string,
+  description: string,
+  createdBy: string,
+  createdAt: string,
+  dueTo: string,
   status: string,
-  priority: number,
-  skipable: string,
   owner: string,
-  created: object,
-  activated: object,
-  processinstanceid: number,
-  processdefinitionid: string,
-  containerid: string,
-  parentid: number,
+  inputData: string,
```

Comment:
not `inputData: PropTypes.shape({})`
and `outputData: PropTypes.shape({})`?

Reply:
rowType isn't needed anymore as the taskList columns will be all dynamic, we can't predict a type for each of them.